stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MediaAttachmentsGroupView.kt

```diff
@@ -30,22 +30,30 @@ import io.getstream.chat.android.ui.message.list.adapter.viewholder.decorator.in
 internal class MediaAttachmentsGroupView : ConstraintLayout {
     var attachmentClickListener: AttachmentClickListener? = null
     var attachmentLongClickListener: AttachmentLongClickListener? = null
-    val maxMediaAttachmentHeight: Int by lazy {
+    private val maxMediaAttachmentHeight: Int by lazy {
         (Resources.getSystem().displayMetrics.heightPixels * MAX_HEIGHT_PERCENTAGE).toInt()
     }
 
     private var state: State = State.Empty
+    private var containerView: View? = null
 
     constructor(context: Context) : super(context.createStreamThemeWrapper())
     constructor(context: Context, attrs: AttributeSet?) : super(context.createStreamThemeWrapper(), attrs)
-    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context.createStreamThemeWrapper(), attrs, defStyleAttr)
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context.createStreamThemeWrapper(),
+        attrs,
+        defStyleAttr)
+
     constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defStyleRes: Int) : super(
         context.createStreamThemeWrapper(),
         attrs,
         defStyleAttr,
         defStyleRes
     )
 
+    fun setMessageContainerView(containerView: View) {
+        this.containerView = containerView
+    }
+
```

Comment:
I just wondering if we could integrate the `setMessageContainerView` into the `setupMediaAttachmentView`. The `setMessageContainerView` must be called before `setupMediaAttachmentView` and it doesn't seem to be used independently.

Reply:
Makes sense